mango-python/bdgenomics/mango/test/variant_test.py

```diff
@@ -40,3 +41,67 @@ def test_visualize_variants(self):
 
         x = variantViz.viewPileup(contig, start, end)
         assert(x != None)
+
+    def test_QualitByDepthDistribution(self):
+        #load file
+        ac = ADAMContext(self.ss)
+        testFile = self.resourceFile("genodata.v3.test.vcf")
+
+        #read variants
+        variants = ac.loadVariants(testFile)
+        _, data = QualityByDepthDistribution(ac, variants).plot(testMode= True)
+
+        expected = Counter({14.6: 10, 12.6: 8, 13.4: 8, 12.9: 7, 15.2: 7, 14.3: 7, 13.2: 7, 11.0: 5, 14.2: 5, \
```

Comment:
Maybe you could just check the first value instead of pasting the whole counter here